frame/support/src/metadata_ir/api.rs

```diff
@@ -0,0 +1,81 @@
+// This file is part of Substrate.
```

Comment:
This could also just be part of `mod.rs`. No  need to have a separate file for this.

Reply:
Indeed api.rs was quite small, have moved that to mod.rs. Thanks!